.travis.yml

```diff
@@ -5,20 +5,34 @@ language: php
 php:
     - 7.2
 
-branches:
-    only:
-        - master
-
-cache:
-    directories:
-        - $HOME/.composer/cache/files
-
 before_install:
-    - phpenv config-rm xdebug.ini
+    - phpenv config-rm xdebug.ini || true
 
 install:
     - travis_retry composer install --classmap-authoritative --no-suggest --prefer-dist
 
 script:
     - vendor/bin/phpunit
-    - vendor/bin/phpcs
+
+jobs:
+    include:
+
+        - stage: Code Quality
+          env: CODING_STANDARDS
+          script:
+              - vendor/bin/phpcs -p
+
+stages:
+    - Test
```

Comment:
You've got `Test` here, but not defined under `.jobs.include[]`. Not sure how to reference the default `.script`, maybe lowercase `test`. I would recommend adding in the another explicit stage, and remove the top level script completely though.

Reply:
`Test` should be the default stage for matrix builds.

If we're considering this a library (https://github.com/libero/schemas/pull/7#discussion_r217739191), we do want a matrix. If not, then we don't and can make them explicit jobs.